packages/components/src/internal/components/domainproperties/DomainForm.spec.tsx

```diff
@@ -1032,4 +1035,18 @@ describe('DomainForm', () => {
 
         expect(form.text()).toContain('Is Primary Key');
     });
+
+    test('with systemFields', () => {
+        const domain = DomainDesign.create({});
+        const form = mount(
+            <DomainFormImpl
+                domain={domain}
+                onChange={jest.fn()}
+                systemFields={SAMPLE_DOMAIN_DEFAULT_SYSTEM_FIELDS}
+                testMode={true}
+            />
+        );
+
+        expect(form.find(SystemFields)).toHaveLength(1);
```

Comment:
should probably add the opposite check in one of the other existing test cases to check that 
`expect(form.find(SystemFields)).toHaveLength(0);`